src/helix-ui/elements/HXSearchAssistanceElement.js

```diff
@@ -1,181 +1,33 @@
-     * @readonly
-     * @type {HTMLElement}
-     */
-    get controlElement () {
-        return this.getRootNode().querySelector(`[aria-controls="${this.id}"]`);
-    }
-
-    /**
-     * Determine if <hx-search-assistance> is visible.
-     *
-     * @default false
-     * @type {Boolean}
-     */
-    get open () {
-        return this.hasAttribute('open');
-    }
-    set open (value) {
-        if (value) {
-            this.setAttribute('open', '');
-        } else {
-            this.removeAttribute('open');
-        }
-    }
-
-    /**
-     * Where to position <hx-search-assistance> in relation to its reference element.
-     *
-     * @default 'bottom-start'
-     * @type {PositionString}
-     */
-    get position () {
-        return this.getAttribute('position') || DEFAULT_POSITION;
-    }
-    set position (value) {
-        if (value) {
-            this.setAttribute('position', value);
-        } else {
-            this.removeAttribute('position');
-        }
-    }
-
-    /**
-     * Reference element used to calculate <hx-search-assistance> position.
-     *
-     * @readonly
-     * @type {HTMLElement}
-     */
-    get relativeElement () {
-        if (this.relativeTo) {
-            return this.getRootNode().getElementById(this.relativeTo);
-        } else {
-            return this.controlElement;
-        }
-    }
-
-    /**
-     * ID of an element relatively to <hx-search-assistance>.
-     *
-     * @type {String}
-     */
-    get relativeTo () {
-        return this.getAttribute('relative-to');
-    }
-    set relativeTo (value) {
-        this.setAttribute('relative-to', value);
-    }
-
-    /** @private */
-    _addOpenListeners () {
-        document.addEventListener('scroll', this._onDocumentScroll);
-        window.addEventListener('resize', this._onWindowResize);
-    }
-
-    /** @private */
-    _attrOpenChange (oldVal, newVal) {
-        let isOpen = (newVal !== null);
-        this.setAttribute('aria-hidden', !isOpen);
-        this.$emit(isOpen ? 'open' : 'close');
-
-        if (isOpen) {
-            this._addOpenListeners();
-            this._reposition();
-        } else {
-            this._removeOpenListeners();
-            this.position = this._initialPosition;
-        }
-    }
-
-    /** @private */
-    _onDocumentScroll () {
-        this._reposition();
-    }
-
-    /** @private */
-    _removeOpenListeners () {
-        document.removeEventListener('scroll', this._onDocumentScroll);
-        window.removeEventListener('resize', this._onWindowResize);
-    }
-
-    /** @private */
-    _reposition () {
-        if (this.relativeElement) {
-            let { x, y } = getPosition({
-                element: this,
-                reference: this.relativeElement,
-                position: this.position,
-            });
-
-            this.style.top = `${y}px`;
-            this.style.left = `${x}px`;
-        }
+        super.$onConnect();
+        this.$defaultAttribute('role', 'search');
```

Comment:
Let's hold off on including the `role` attr. This means we'll have to remove the whole `$onConnect` method.